### PR TITLE
Disable RegisterNow Button of all Foundation Courses

### PR DESCRIPTION
**Pull request by vasudev19126**

### CODE REVIEW

File: src/views/pages/Course/ApplyModal.js [hunk 1]
```diff
@@ -116,16 +117,31 @@ export default function ApplyModal({
 
   return (
     <div>
-      <Button
+      {isFoundation ? (
```
Comment (Abhishek-kumar09) [inline src/views/pages/Course/ApplyModal.js:120]: Why isFoundation?

It is looking like you have disbled the button for course, but we can have a more general approach to it.

Suppose in future we want one of the Training course to not accept responses, then this apporach will not work there

File: src/views/pages/Course/ApplyModal.js [hunk 2]
```diff
@@ -126,6 +127,20 @@ export default function ApplyModal({
...
+        variant="contained"
+        onClick={handleClickOpen}
+        {...rest}
+        fullWidth={fullWidth}
+        disabled
+      >
+        Register Now
+      </Button>
+      )}
+
```
Comment (Abhishek-kumar09) [inline src/views/pages/Course/ApplyModal.js:143]: You could have written:
```javascript
        <Button
        className={classes.btn}
        size="large"
        variant="contained"
        onClick={handleClickOpen}
        {...rest}
        fullWidth={fullWidth}
        disabled={batch.active} <------------
      >
        Register Now
      </Button>
```

Comment (vasudev19126) [reply src/views/pages/Course/ApplyModal.js:143]: okay 👍

**Review by Abhishek-kumar09 — COMMENTED** [review]
image

Button is disabled but is not visible, can we do something else, like "Course is not accepting applications" 
The UI needs some improvements here.

**Comment by vasudev19126** [comment]
> image
> 
> Button is disabled but is not visible, can we do something else, like "Course is not accepting applications"
> The UI needs some improvements here.

Okay, I'll look into that.

**Pull request merged**